lib/instrumentation/grpc-js/grpc.js

```diff
@@ -202,6 +205,14 @@ function wrapRegister(shim, original) {
   }
 }
 
+function isGrpcError(statusCode, config) {
```

Comment:
Whatcha think about changing the name of this function, like `shouldTrackError` or something to that effect? Reason I ask is because even when this returns `false`, they're still GRPC errors, we just don't care about them, so it's a bit misleading

Reply:
Makes sense, will do.